pkgs/development/interpreters/lfe/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, erlang, makeWrapper, coreutils, bash }:
+
+with stdenv.lib;
```

Comment:
Outside of the meta attribute set this doesn't seem to be used. Maybe move it there for more specificity?